src/Features/Core/Portable/Completion/CompletionItem.cs

```diff
@@ -211,6 +264,23 @@ public sealed class CompletionItem : IComparable<CompletionItem>
             string? displayTextSuffix = null,
             string? inlineDescription = null,
             bool isComplexTextEdit = false)
+        {
+            return CreateInternal(
+                displayText, filterText, sortText, properties != null ? properties.ToImmutableArray() : default, tags, rules, displayTextPrefix,
+                displayTextSuffix, inlineDescription, isComplexTextEdit);
+        }
+
+        internal static CompletionItem CreateInternal(
```

Comment:
>CreateInternal [](http://example.com/codeflow?start=39&length=14)

I really wanted to just call this Create to match the other calls, but the compiler would have none of that unless I reordered or added a param.